rero_ils/modules/users/jsonschemas/users/user-v0.0.1.json

```diff
@@ -103,6 +104,27 @@
         ]
       }
     },
+    "username": {
+      "title": "Username",
+      "description": "Login username for the web interface.",
+      "type": "string",
+      "pattern": "^[a-zA-Z0-9][a-zA-Z0-9-_]{2}[a-zA-Z0-9-_]*$",
```

Comment:
Should this not be the same as `ACCOUNTS_USERNAME_REGEX=r"^[a-zA-Z0-9-_]{2,255}$"`?